packages/wallets/src/features/accounts/screens/ResubmitPOA/ResubmitPOA.tsx

```diff
@@ -16,33 +20,102 @@ const ResubmitPOA: React.FC = () => {
     };
 
     return (
-        <div className='wallets-resubmit-poa'>
-            <div className='wallets-resubmit-poa__address'>
-                <div className='wallets-resubmit-poa__address--title'>
+        <div className='wallets-poa'>
+            <div className='wallets-poa__address'>
+                <div className='wallets-poa__address__title'>
                     <WalletText weight='bold'>Address</WalletText>
-                    <div className='wallets-resubmit-poa__address--title-divider' />
+                    <div className='wallets-poa__address__title__divider' />
                 </div>
-                <div className='wallets-resubmit-poa__address--inline'>
+                <div className='wallets-poa__address__container__inline'>
                     <InlineMessage size='md' type='warning' variant='contained'>
-                        <div className='wallets-resubmit-poa__address--inline-message'>
+                        <div className='wallets-poa__address__inline-message'>
                             For faster verification, input the same address here as in your proof of address document
                             (see section below)
                         </div>
                     </InlineMessage>
                 </div>
-                <div className='wallets-resubmit-poa__address--input'>
-                    <WalletTextField label='First line of address*' name='first-line' />
-                    <WalletTextField label='Second line of address' name='second-line' />
-                    <WalletTextField label='Town/City*' name='town-line' />
-                    <WalletDropdown
-                        label='State/Province'
-                        list={statesList}
-                        listHeight='sm'
-                        name='wallets-resubmit-poa__dropdown'
-                        onSelect={handleSelect}
-                        value={selectedState}
-                    />
-                    <WalletTextField label='Postal/ZIP Code' name='zip-line' />
+                <div className='wallets-poa__address__input'>
+                    <FlowTextField label='First line of address*' name='first-line' />
```

Comment:
Can we use camelCase for the names? Like `firstLine` or `secondLine` cuz this will be matched within the validation schema like:

<img width="869" alt="Screenshot 2023-11-17 at 2 53 15 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/b59f1e09-a97f-4d49-831e-02c59025c6ca">

Reply:
can can